experiments/Atmos_GCM/heldsuarez.jl

```diff
@@ -21,17 +21,19 @@ function init_heldsuarez!(bl, state, aux, coords, t)
 
     FT = eltype(state)
 
-    # TODO: change to altitude function
-    r = norm(coords, 2)
-    h = r - FT(planet_radius)
-    # h = altitude(bl.orientation, aux)
+    h = altitude(bl.orientation, aux)
+    e_pot = gravitational_potential(bl.orientation, aux)
 
     scale_height = R_d * FT(T_initial) / grav
     p            = FT(p_ground) * exp(-h / scale_height)
 
-    state.ρ  = air_density(FT(T_initial), p)
+    ts = PhaseDry_given_pT(p, FT(T_initial))
+    e_int = internal_energy(ts)
+    ρ = air_density(ts)
+
+    state.ρ  = ρ
     state.ρu = SVector{3, FT}(0, 0, 0)
-    state.ρe = state.ρ * (internal_energy(FT(T_initial)) + aux.orientation.Φ)
+    state.ρe = state.ρ * (e_int + e_pot)
```

Comment:
Need to add noise (e.g., in energy or density) to break symmetry of initial condition. See @bischtob’s HS branch.

Reply:
I'll leave this for #746, since this PR currently should not change behavior.